### DISCUSSION TITLE
Shipping costs within an installation with more than 1 portal

### DISCUSSION THREAD

**Discussion by Rodien** [openstore-ecommerce/OpenStore#212] [Q&A]
Hi guys,
  
Shipping costs does not work when working with 2 sites in one installation. 

I think that it has something to do with cache because when I am let's say on site1 and I restart the app 
then it starts working but then it does not work on site2 and vice versa.

I'm using Open Store 4.1.3 and NBrightTemplateSys 8.6.2. 

Is this a known issue?

**Comment by DNNMonster** [comment]
I am not currently working with multiple portal installs so I had not seen this before.  It will be either the cache or something else gone astray in the code.  You're description sure makes it look like there is a portal id issue somewhere in the flow.  Maybe just a template cache key...
